api/src/main/java/io/minio/messages/AssumeRoleWithClientGrantsResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.minio.messages;
+
+import org.simpleframework.xml.Element;
+import org.simpleframework.xml.Namespace;
+import org.simpleframework.xml.Root;
+
+@Root(name = "AssumeRoleWithClientGrantsResponse", strict = false)
+@Namespace(reference = "https://sts.amazonaws.com/doc/2011-06-15/")
+public class AssumeRoleWithClientGrantsResponse {
+
+  @Element(name = "AssumeRoleWithClientGrantsResult")
+  private AssumeRoleWithClientGrantsResult clientGrantsResult;
+
+  public AssumeRoleWithClientGrantsResult getClientGrantsResult() {
```

Comment:
You could have below instead of this method.
```java
public Credentials credentials() {
  return clientGrantsResult == null ? null : clientGrantsResult.credentials();
}
```
}

Reply:
1. i'm not sure, i'll take a look at wednesday or sunday, ok?
2. my bad